open-api/rest-catalog-open-api.py

```diff
@@ -1103,6 +1103,7 @@ class LoadTableResult(BaseModel):
     ## General Configurations
 
     - `token`: Authorization bearer token to use for table requests if OAuth2 security is enabled
+    - `expires-at-ms`: if present, specifies timestamp in milliseconds when credentials for storage(AWS S3/Azure/GCP), specified in config would expire
```

Comment:
I don't think it's actually correct to have this at this level. This should be specific to the underlying Storage provider. GCS for example has `gcs.oauth2.token-expires-at`. That being said, the respective FileIO properties of each storage provider should have a similar property to the one used for GCS

Reply:
Based on https://github.com/apache/iceberg/pull/10722#discussion_r1796948974, I was thinking to move/add this to fileIo Properties once linked PRs in the description are merged
I saw this [comment](https://github.com/apache/iceberg/pull/11281#discussion_r1804199315), where it was decided not having the separate property. 
It would be beneficial to have this property but will yet to go through the recording of Catalog sync (recording is not yet [available](https://docs.google.com/document/d/1iPGVCIcr-M0XtAiudOguWAvmqIdVgpYN5vz5ohO8PKw/edit?tab=t.0))

Reply:
Yes, the idea is to move the expiration setting to the respective FileIO implementation rather than having a separate field in the OpenAPI spec